### PR TITLE
fix: move all fixtures from jsonl to parquet

### LINKED ISSUE #2580: Move all fixtures off of jsonl
JSONL export is hard to re-create as we no longer have our own tracer. We should store fixtures as something that works for demo data, local development.

### PR DESCRIPTION
**Pull request by saikolasani**
Closes #2580

### CODE REVIEW

**Comment by saikolasani** [comment]
recheck

**Comment by saikolasani** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by saikolasani** [comment]
recheck

**Comment by saikolasani** [comment]
I need to change random.jsonl to random.parquet. It should be done otherwise.

**Comment by saikolasani** [comment]
Need to finish testing but basic tests show same functionality as before

File: src/phoenix/trace/fixtures.py [hunk 1]
```diff
@@ -200,13 +200,16 @@ def get_trace_fixture_by_name(fixture_name: str) -> TracesFixture:
...
+    bucket: Optional[str] = "arize-phoenix-assets",
+    prefix: Optional[str] = "traces/",
 ) -> List[str]:
     """
     Downloads the traces fixture from the phoenix bucket.
     """
     url = f"{host}{bucket}/{prefix}{fixture.file_name}"
+    if is_parquet_file(url):
+        return [url]
+
```
Comment (axiomofjoy) [inline src/phoenix/trace/fixtures.py:212]: `download_traces_fixture` is not downloading the data for Parquet files, the download is happening in `load_example_traces`. Can we rearrange things so the name of the function more accurately reflects where the download is happening? Both functions may no longer be necessary.

File: src/phoenix/trace/fixtures.py [hunk 2]
```diff
@@ -197,26 +196,21 @@ def get_trace_fixture_by_name(fixture_name: str) -> TracesFixture:
...
-        return cast(List[str], f.readlines())
-
-
 def load_example_traces(fixture_name: str) -> TraceDataset:
     """
     Loads a trace dataframe by name.
     """
+    host = ("https://storage.googleapis.com/",)
+    bucket = ("arize-phoenix-assets",)
+    prefix = ("traces/",)
```
Comment (axiomofjoy) [inline src/phoenix/trace/fixtures.py:205]: Can we make these strings rather than tuples of strings?

File: src/phoenix/trace/utils.py [hunk 3]
```diff
@@ -1,11 +1,34 @@
...
+    """
+    Check if the given file is a Parquet file.
+    """
+    file_extension = os.path.splitext(file_path)[-1]
+    if file_extension == ".jsonl":
+        return True
+    return False
+
+
+def store_json_traces_fixture(
```
Comment (axiomofjoy) [inline src/phoenix/trace/utils.py:21]:
```suggestion
def download_json_traces_fixture(
```

"store" suggests this function persists the data somehow.

**Review by axiomofjoy — APPROVED** [review]

**Pull request merged**